src/main/java/com/srans/nestserver/model/Invoice.java

```diff
@@ -1,200 +1,265 @@
 package com.srans.nestserver.model;
-import java.io.Serializable;
 
-import javax.persistence.Column; 
+import javax.persistence.Column;
 import javax.persistence.Entity;
-import javax.persistence.GeneratedValue; 
+import javax.persistence.GeneratedValue;
 import javax.persistence.GenerationType;
-import javax.persistence.Id; 
+import javax.persistence.Id;
 import javax.persistence.Table;
 
-import org.hibernate.annotations.Formula;
 
 @Entity
 
 @Table(name = "invoice")
```

Comment:
Do we need table for this?
I don't think we should save data, it's just a bean for placeholder, we no need to store it on database

Reply:
Sir, you are right, but this table has been made by likhit and has also made some api, if you say I will delete